### DISCUSSION TITLE
Unsupported: 915005997801 Philips Hue Xamento

### DISCUSSION THREAD

**Discussion by caveman2st** [Koenkk/zigbee2mqtt#11631] [Devices]
Hello,
please can you add this new Philips Hue Device:

{"id":9,"type":"Router","ieeeAddr":"0x001788010b4190c3","nwkAddr":41900,"manufId":4107,"manufName":"Philips","powerSource":"Mains (single phase)","modelId":"915005997801","epList":[11,242],"endpoints":{"11":{"profId":260,"epId":11,"devId":269,"inClusterList":[0,3,4,5,6,8,4096,768,64513],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"915005997801","manufacturerName":"Philips","powerSource":1,"zclVersion":2,"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210305","swBuildId":"1.82.10"}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210305","swBuildId":"1.82.10","zclVersion":2,"interviewCompleted":true,"meta":{},"lastSeen":1645954932568,"useImplicitCheckin":true}

I think, it's similar to [929002966401] https://www.zigbee2mqtt.io/devices/929002966401.html

Best regards
Stefan

**Comment by Koenkk** [comment]
Please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+

**Reply by caveman2st** [reply]
done
